R/autoplot.R

```diff
@@ -2,11 +2,10 @@
 #'
 #' @description
 #'
-#'  This function generates a complete 'ggplot' object suitable for
-#'  visualizing time series data in an `aedseo_tsd` object. It creates a line
-#'  plot connecting the observations and adds points at each data point.
+#'  This function generates a complete 'ggplot' object suitable for visualizing time series data in an `tsd` object.
+#' It creates a line plot connecting the observations and adds points at each data point.
 #'
-#' @param object An `aedseo_tsd` or `aedseo` object
+#' @param object An `tsd` or `seasonal_onset_tsd` object
```

Comment:
Consider just having this as a `tds` object and then multiclassing the `seaonal_onset_tds` objects as both `seaonal_onset_tds` and `tsd`.

This is analog to `tibble` and `data.frame`. If it works for a `data.frame` you can also supply a `tibble` but there is no need to write that both are accepted.

Reply:
Also, isn't it just called "seasonal_onset" in the code?